pages/docs/couple-your-code/couple-your-code-initializing-coupling-data.md

```diff
@@ -54,3 +36,9 @@ Now, you can specify at runtime if you want to initialize coupling data. For exa
 <exchange data="Displacements" mesh="StructureMesh" from="SolidSolver" to="FluidSolver" initialize="yes"/>
 [...]
 ```
+
+If you are using a serial coupling scheme, only initial data that is written from the second to the first participant is actually used. Initial data written from the first to the second participant will be ignored, because in a serial coupling scheme the first participant already computes its first result before the second participant even starts.
+
+{% note %}
+preCICE still supports data initialization for both participants, even if a serial coupling scheme is used. You can read our section on [time interpolation](couple-your-code-waveform) to learn more about the reasons.
+{% endnote %}
```

Comment:
I had to read this part three times before understanding it. Could we improve it?
Instead of "written from A to B", I would always say "written to A and then communicated by preCICE from A to B".
Either all in a note or non. Remember that waveform will be the norm with v3 eventually.

Reply:
No so sure about this whole paragraph anymore, because it makes things more complicated and actually we are now planning to use initial data for every participant for every coupling scheme. Also the second participant in a serial coupling scheme uses the initial data it receives from the first. Even for waveform order = 0 we use the initial data for the relative time = 0. This is a very narrow edge case, but it leads to a consistent treatment across windows. See also the following figure:

![Screenshot from 2023-05-28 14-11-10](https://github.com/precice/precice.github.io/assets/5740604/2d424c5b-e724-476d-9074-f90148fa46f4)

How about removing this paragraph and opening the whole topic later when we introduce time interpolation?

Reply:
We could also use the figure from above later in the waveform relaxation section, if it is useful. For the section on data initialization, I think it contains too much information that has not been introduced yet.

Reply:
See [ec217b5](https://github.com/precice/precice.github.io/pull/186/commits/ec217b58ee30129a1d940147c7b2299b934af163) for some steps into this direction.